packages/react-storage/src/components/StorageBrowser/context/locationActions/types.ts

```diff
@@ -34,8 +34,8 @@ export interface LocationActions<T = Permission> {
 export type LocationActionsAction<T = string> =
   | { type: 'CLEAR' }
   | { type: 'SET_ACTION'; payload: T }
-  | { type: 'SET_LOCATION_ITEM'; item: LocationItem }
-  | { type: 'UNSET_LOCATION_ITEM'; key: string };
+  | { type: 'TOGGLE_SELECTED_ITEM'; item: LocationItem }
+  | { type: 'TOGGLE_SELECTED_ITEMS'; items: LocationItem[] | [] };
```

Comment:
Think this action can be defined as:
```ts
  | { type: 'TOGGLE_SELECTED_ITEMS'; items?: LocationItem[] };
```
where passing `undefined` resets the selected items?

Reply:
oh nice, I like it. Will update